recipe/patch_yaml/pytorch.yaml

```diff
@@ -160,3 +160,56 @@ if:
 
 then:
   - add_constrains: pybind11 <0.0.0a0
+
+
+---
+# Pytorch 2.4.1 did not upload a package for pytorch-gpu and pytorch-cpu that
+# compatible with all versions of python
+# Only the linux builds use the "megabuild" that cause the issue
+# The OSX builds were "safe"
+# https://github.com/conda-forge/pytorch-cpu-feedstock/issues/338
+# https://github.com/conda-forge/pytorch-cpu-feedstock/pull/341
+
+if:
+  name: pytorch-gpu
+  version: 2.4.1
+  subdir_in: linux-64
+  build_number_in:
+    - 0
+    - 1
+    - 2
+    - 3
+    - 4
+    - 5
+    # mkl is +100
+    - 100
+    - 101
+    - 102
+    - 103
+    - 104
+    - 105
+    # cuda compiler is +200
+    - 200
+    - 201
+    - 202
+    - 203
+    - 204
+    - 205
+    # cuda compiler + mkl is +300
+    - 300
+    - 301
+    - 302
+    - 303
+    - 304
+    - 305
+
+then:
+  - replace_depends:
+      old: "pytorch 2.4.1=cuda118_*"
+      new: "pytorch 2.4.1=cuda118_*${build_number}"
```

Comment:
where do you expect `${build_number}` to come from? I'd be surprised if this works. I'd also be fine to just fix the `**5` build numbers.

Reply:
https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/blob/f43ade775561f5fbc919ef47b3942f937821e8cb/recipe/patch_yaml_utils.py#L14

Reply:
OK great, I didn't know! Could you post the result of `show_diff.py`?

Reply:
yeah the CPU stuff is wrong, fixing that now ;)